README.md

```diff
@@ -11,7 +11,7 @@ Documentation to give to partners who would like to integrate with our API.
 Setting up ruby (only need to be done once per machine):
 
     curl -sSL https://get.rvm.io | bash -s stable
-    rvm install ruby-2.4.1
+    rvm install ruby-2.3.1
```

Comment:
This seems to be different than the required ruby version specified in the gem file.